control-plane/roles/gardener/defaults/main/main.yaml

```diff
@@ -0,0 +1,94 @@
+---
+gardener_image_vector_overwrite:
+gardener_component_image_vector_overwrite:
+
+gardener_extension_provider_metal_repo_ref: "{{ gardener_extension_provider_metal_image_tag }}"
+gardener_os_controller_repo_ref: "{{ gardener_os_controller_image_tag }}"
+# TODO: as soon as our changes on cilium controller are merged to upstream, we can remove the path to the fork
+gardener_networking_cilium_repo_ref: "metal-stack/gardener-extension-networking-cilium/{{ gardener_networking_cilium_image_tag }}"
```

Comment:
Which changes? Is there a PR we could link?

Reply:
I improved the details here a bit when we can remove this override.